lib/api/v3/activities/activity_representer.rb

```diff
@@ -99,7 +96,7 @@ def initialize(model, options = {})
                    html_details = render_details(represented)
                    formattables = details.zip(html_details)
 
-                   formattables.map { |d| { format: 'textile', raw: d[0], html: d[1] } }
+                   formattables.map { |d| { format: 'custom', raw: d[0], html: d[1] } }
```

Comment:
@NobodysNightmare if 'custom' is used in the api as a format, it should be documented in the specs as well.